automation/bumper-bot/bumper.go

```diff
@@ -0,0 +1,104 @@
+//usr/bin/env go run $0 "$@"; exit
```

Comment:
We should keep this program under /tool. Automation then should contain any glue needed to be able to run it on an arbitrary CI host.

All the scripts should be available to humans. Automation directory is just to proxy for CI.